ui/lib/core/addon/components/search-select-with-modal.hbs

```diff
@@ -82,17 +82,15 @@
   {{/if}}
 
   {{! wait until user has selected 'create a new item' before rendering modal}}
-  {{#if this.nameInput}}
-    <Modal
-      @title="Create new {{singularize @id}}"
-      @onClose={{action (mut this.showModal) false}}
-      @isActive={{this.showModal}}
-      @type="info"
-      @showCloseButton={{false}}
-    >
-      <section class="modal-card-body">
-        {{#if @modalSubtext}}
-          <p class="has-bottom-margin-s" data-test-modal-subtext>
+  {{#if (and this.nameInput this.showModal)}}
+    <Hds::Modal id="form-modal" @onClose={{fn (mut this.showRotationModal) false}} as |M|>
```

Comment:
```suggestion
    <Hds::Modal id="form-modal" @onClose={{fn (mut this.showModal) false}} as |M|>
```
off the bat comment: should this be `showModal`? to match the boolean variable we use to show/hide the modal

Reply:
Yep - this was copy pasta! I actually just noticed it earlier this morning when checking out the branch for the first time in a while. Funny timing!

Reply:
yeah, I doubt that solves any of the main issues here, so let's chat in a bit